src/pages/rainyday/index.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+
+import Navbar from '../../components/Navbar';
+import Content from '../../components/Content';
+import Text from '../../components/Text';
+import Footer from '../../components/Footer';
+
+import {
+  // BackgroundEffect,
```

Comment:
```suggestion
```

Reply:
esse é componente estilizado que não consegui utilizar, porque a animação não funciona :sad_pepe: 

será que vale a pena remover enquanto não for chamado? 🤔